biogeochem/FatesLandUseChangeMod.F90

```diff
@@ -30,7 +31,8 @@ module FatesLandUseChangeMod
   public :: get_landuse_transition_rates
```

Comment:
We've generally had the convention of CamelCase() for procedures in FATES.

Reply:
_[Pushes glasses up nose]_ Technically that's PascalCase!